types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts

```diff
@@ -49,14 +49,14 @@ configuration = {
 
 configuration = {
     // ...
-    plugins: [new MiniCssExtractPlugin()],
+    plugins: [new MiniCssExtractPlugin({})],
```

Comment:
Why this change? If you aren't configuring anything shouldn't `MiniCssExtractPlugin()` be valid?

Reply:
oh, this is just kind of typo, I was under impression the parameterless ctro is already ther in thest,
to be amended. thx!